lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -63,6 +63,8 @@ namespace Js
     const int magicEndOfAsmJsModuleInfo = *(int*)"]asmmodinfo";
     const int magicStartOfPropIdsOfFormals = *(int*)"propIdOfFormals[";
     const int magicEndOfPropIdsOfFormals = *(int*)"]propIdOfFormals";
+    const int magicStartOfSlotIdToNestedIndexArray = *(int*)"slotIdToNestedIndexArray[";
```

Comment:
Lol, this magic part is not what it's supposed to be :) your usage accidentally works- the dbgscope ones dont- the magic reinterprets the first 4 chars of a string as a uint32 so you really want only 4 char strings- in the case of magicStartOfDebuggerScopes and magicStartOfDebuggerScopeProperties, the first 4 chars are the same- really all of these longer than 4 chars should be fixed to be unique 4 char strings (3 + [)